src/core_modules/capture-core/rulesEngineActionsCreator/converters/momentConverter.js

```diff
@@ -1,7 +1,8 @@
 // @flow
 /* eslint-disable class-methods-use-this */
 import moment from 'moment';
-import type { IMomentConverter } from '../RulesEngine/rulesEngine.types';
+import { getFormattedStringFromMomentUsingEuropeanGlyphs } from 'capture-core-utils/date';
+import type { IMomentConverter } from 'capture-core-utils/RulesEngine/rulesEngine.types';
 
 const momentFormat = 'YYYY-MM-DD';
```

Comment:
so if we decide to change the signature and add the default values we wont need to pass this value here any more 👯

Reply:
👍